params/governance_paramset_test.go

```diff
@@ -351,6 +351,7 @@ func TestGovParamSet_ChainConfig(t *testing.T) {
 				GasTarget:                 30000000,
 				MaxBlockGasUsedForBaseFee: 60000000,
 				BaseFeeDenominator:        20,
+				Kip82Ratio:                "20/80",
```

Comment:
It may be changed as default

Reply:
It's reverted back to 20/80 in 0b3e45e

Reply:
You can check https://github.com/klaytn/klaytn/pull/1631#discussion_r1024772248